userapi/consumers/roomserver_test.go

```diff
@@ -61,13 +61,13 @@ func Test_evaluatePushRules(t *testing.T) {
 		}{
 			{
 				name:         "m.receipt doesn't notify",
-				eventContent: `{"type":"m.receipt"}`,
+				eventContent: `{"type":"m.receipt","sender":"@test:remotehost"}`,
```

Comment:
One of my iterations of changes required this. It is no longer required, which is comforting since the existing behaviour must be being maintained now.